BabelfishDump.spec

```diff
@@ -20,8 +20,9 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 
 %define _trivial .0
-%define _buildid .1
+%define _buildid .2
 
+%define _build_id_links none
```

Comment:
Actually, I was wrong. You probably want to leave the `.buildid` items in your RPM. [Reference for the utility of these files](https://fedoraproject.org/wiki/Releases/FeatureBuildId).